go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/blocknative/dreamboat
 
-go 1.18
+go 1.19
```

Comment:
👀

Reply:
@aratz-lasa I had a thought that we'll be testing that anyway :D